pkg/cloud-controller-manager/loadbalancer.go

```diff
@@ -0,0 +1,371 @@
+package ccm
```

Comment:
Inconsistent style of test filenames

loadbalancer.go and loadBalancer_test.go